qtop.py

```diff
@@ -900,7 +932,12 @@ def exec_func_tuples(func_tuples):
 
     cwd = os.getcwd()
     QTOPPATH = os.path.expanduser(cwd)
-    config = load_yaml_config(QTOPPATH)
+    USERPATH = os.path.expandvars('$HOME/.local/qtop')
+    try:
+        config = load_yaml_config(USERPATH)
+    except IOError:
+        config = load_yaml_config(QTOPPATH)
```

Comment:
for the record: this configuration is only an if-else situation; original qtop could "merge" the config vars across multiple configs; this is possibly a feature request